lib/polyfill/storage_estimate.js

```diff
@@ -0,0 +1,54 @@
+/*! @license
+ * Shaka Player
+ * Copyright 2016 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+goog.provide('shaka.polyfill.StorageEstimate');
+
+goog.require('shaka.polyfill');
+
+
+/**
+ * @summary A polyfill to provide navigator.storage.estimate in old
```

Comment:
Since this is almost exactly the code from https://developers.google.com/web/updates/2017/08/estimating-available-storage-space#the-present, please cite that article here.  That article's code is licensed under Apache 2.0 in the footer, so it is okay to adopt, but I think it would still be best to cite where it came from.

Reply:
Done!